docker-compose.yml

```diff
@@ -39,3 +39,5 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_DB: postgres
       POSTGRES_PASSWORD: docker
+    ports:
+      - 5432:5432
```

Comment:
For local development

Reply:
This mapping needs to be careful because Postgres is pretty common on local machines and they are already using 5432. In that scenario, the service will fail to start up.

So sometimes people will purposely use other port on host, e.g. `5999:5432`. But in general this is fine.